tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "skipLibCheck": true,
```

Comment:
@nandubatchu pls check and verify implications of this

Reply:
looks fine as per the comment from SO - https://stackoverflow.com/questions/52311779/usage-of-the-typescript-compiler-argument-skiplibcheck